### PR TITLE
Add 2.x-centos7 image

### PR DESCRIPTION
**Pull request by olegrok**

### CODE REVIEW

File: 2.x-centos7/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+MAINTAINER mail@racktear.com
+
+RUN groupadd tarantool \
+    && adduser -g tarantool tarantool
+
+ENV TARANTOOL_VERSION=2.1.2-9-g72394afda \
+    TARANTOOL_DOWNLOAD_URL=https://github.com/tarantool/tarantool.git \
+    TARANTOOL_INSTALL_LUADIR=/usr/local/share/tarantool \
+    CURL_REPO=https://github.com/curl/curl.git \
+    CURL_TAG=curl-7_64_0 \
```
Comment (Totktonada) [inline 2.x-centos7/Dockerfile]: Tarantool works correctly with libcurl 7.60 and above after 2.1.2-49-gc413011cc when threaded DNS resolver is enabled (it is default). Please, update the tarantool version (it is better option) or use curl-7.59.

Comment (olegrok) [reply 2.x-centos7/Dockerfile]: Updated to 2.2.0-240-gfa6bb2fe0

Comment (Totktonada) [reply 2.x-centos7/Dockerfile]: Are you sure you want to try an unreleased version? I would suggest to put last 2.1 here. At the time of writing it is 2.1.2-52-g154345fc5.

Comment (olegrok) [reply 2.x-centos7/Dockerfile]: Ok. 2.1.2-52-g154345fc5

**Comment by Totktonada** [comment]
Added 2.x-centos7 tag to docker hub (will built within a day).

**Pull request merged**